src/main/java/io/temporal/internal/metrics/MetricsType.java

```diff
@@ -42,13 +42,19 @@
       TEMPORAL_METRICS_PREFIX + "workflow_task_schedule_to_start_latency";
   public static final String WORKFLOW_TASK_EXECUTION_LATENCY =
       TEMPORAL_METRICS_PREFIX + "workflow_task_execution_latency";
+  // Total latency of a workflow task which can include multiple synchronous decision tasks
```

Comment:
even with the comment, it is still not obvious to me what the difference is between execution latency and execution total latency?

Reply:
[Nit] also styling question on when to use "//" comment vs "/** */" comment delimiters, as is done below on lines 48, 51

Reply:
Fixed comments.

To understand this comment you need to know about workflow task heartbeat feature. Basically a workflow task when reporting completion can set ForceCreateNewWorkflowTask property to true on RespondWorkflowTaskCompletedRequest. In this case, the new workflow task is created immediately and returned as a response to the RespondWorkflowTaskCompletedRequest call. Such call acts as a heartbeat that extends the duration of a workflow task. Thus a single total execution latency of a workflow task can be larger than latencies of multiple such synchronously created tasks.